components/global/at_mentions_button/at_mentions_button.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Tooltip} from 'react-bootstrap';
+import {FormattedMessage} from 'react-intl';
+
+import IconButton from '@mattermost/compass-components/components/icon-button';
+
+import OverlayTrigger from 'components/overlay_trigger';
+import Constants, {RHSStates} from 'utils/constants';
+
+type Props = {
+    rhsState: typeof RHSStates[keyof typeof RHSStates] | null;
+    isRhsOpen?: boolean;
+    actions: {
+        showMentions: () => void;
+        closeRightHandSide: () => void;
+    };
+};
+
+const AtMentionsButton = (props: Props): JSX.Element => {
+    const {rhsState, isRhsOpen, actions: {closeRightHandSide, showMentions}} = props;
+
+    const mentionButtonClick = (e: React.MouseEvent<HTMLButtonElement>) => {
+        e.preventDefault();
+        if (rhsState === RHSStates.MENTION) {
+            closeRightHandSide();
+        } else {
+            showMentions();
+        }
+    };
+
+    const tooltip = (
+        <Tooltip id='recentMentions'>
+            <FormattedMessage
+                id='channel_header.recentMentions'
+                defaultMessage='Recent mentions'
+            />
+        </Tooltip>
+    );
+
+    return (
+        <OverlayTrigger
+            trigger={['hover']}
+            delayShow={Constants.OVERLAY_TIME_DELAY}
+            placement='bottom'
+            overlay={isRhsOpen ? <></> : tooltip}
```

Comment:
That is a good question for UX as I just copied the previous behaviour. Maybe @michaelgamble can advice?

Reply:
I think that may have been related to an issue that came up a while back where tooltips overlapped menus, this may have been the approach to fix it back then.

I agree that it would be good to still have tooltips regardless of whether the RHS is open.

@nevyangelova, I would say that we can remove this in the Global Header (leave whatever functionality is there when the GH is off), but that it can be done with the next PR as it's not functionally critical for this PR.